packages/client/src/components/variables.scss

```diff
@@ -18,6 +18,7 @@ $default-font-family: 'Lato', sans-serif;
 $logo-font-family: 'Shadows Into Light', cursive;
 
 //--- Font Sizes ---//
+$huge-font-size: 8rem;
```

Comment:
LOL accepting a better name than this.

Reply:
$godzilla-font-size

Reply:
I think i'm actually gonna take godzilla LMAO

Reply:
lol let's do it